.github/workflows/test.yml

```diff
@@ -144,8 +144,8 @@ jobs:
         mypy -c "from tinygrad.tensor import Tensor; print(Tensor([1,2,3,4,5]))"
     - name: Test DEBUG
       run: DEBUG=100 python3 -c "from tinygrad import Tensor; N = 1024; a, b = Tensor.rand(N, N), Tensor.rand(N, N); c = (a.reshape(N, 1, N) * b.T.reshape(1, N, N)).sum(axis=2); print((c.numpy() - (a.numpy() @ b.numpy())).mean())"
-    - name: Repo line count <9500 lines
-      run: MAX_LINE_COUNT=9500 python sz.py
+    - name: Repo line count <9600 lines
+      run: MAX_LINE_COUNT=9600 python sz.py
```

Comment:
TBH this requirement is kind of silly since the code has many lines that are way too long and do not adhere to any standard the Python community agreed on.